README.md

```diff
@@ -221,27 +234,31 @@ Swept-tuned detection is a standard spectrum analyzer measurement. The `SweptTun
 |`reference`|false|string|N/A|Data reference point, e.g., `"receiver input"`, `"antenna output"`, `"output of isotropic antenna"`.|
 
 ##### YFactorCalibration Object
-The `YFactorCalibration` object requires the following:  
+The purpose of the `YFactorCalibration` is to provide parameters and results for time-domain y-factor calibrations. The `YFactorCalibration` object contains the following name/value pairs:  
 
 |name|required|type|unit|description|
 |----|--------------|-------|-------|-----------|
 |`last_time_performed`|true|datetime|[ISO-8601](https://github.com/gnuradio/SigMF/blob/master/sigmf-spec.md#the-datetime-pair)|Date and time of last calibration.|
-|`calibrations`|false|array|dB|Receiver attenuations and corresponding gain and noise figure arrays equal in length to the [`sample_count`](https://github.com/gnuradio/SigMF/blob/master/sigmf-spec.md#annotation-segment-objects).|    
+|`frequency`|false|array|Hz|Frequencies that y-factor calibrations are performed.|
+|`excess_noise_ratio`|false|array|dB|Excess noise ratio of calibrated noise source at `frequency` of y-factor calibration.|
+|`receiver_setting_name`|false|string|N/A|Name of receiver setting. Use if y-factor calibration is performed over a range of a receiver parameter, e.g., `"attenuation"`, `"input range"`.|
+|`receiver_setting_units`|false|string|N/A|Units corresponding to `receiver_setting_name`, e.g., `"dB"`, `"dBm"`.|
 |`reference`|false|string|N/A|Data reference point, e.g., `"receiver input"`, `"antenna output"`, `"output of isotropic antenna"`.|
+|`calibrations`|false|array|dB|Receiver settings and corresponding gain and noise figure arrays equal in length to the length of `frequency`.|
 
-Example of `calibrations`:
+Example of `calibrations` for two receiver settings and ten frequencies:
 
 ```
 [
   {
-    "attenuation": 1,
-    "gains": [],
-    "noise_levels": []
+    "receiver_setting": 1,
+    "gain": [7.1, 7.2, 7.0, 7.1, 7.4, 7.1, 7.0, 7.3, 7.2, 7.0],
+    "noise_figure": [9.1, 9.2, 9.0, 9.1, 9.4, 9.1, 9.0, 9.3, 9.2, 9.0]
   },
   {
-    "attenuation": 2,
-    "gains": [],
-    "noise_levels": []
+    "receiver_setting": 2,
+    "gain": [6.1, 6.2, 6.0, 6.1, 6.4, 6.1, 6.0, 6.3, 6.2, 6.0],
+    "noise_figure": [9.1, 9.2, 9.0, 9.1, 9.4, 9.1, 9.0, 9.3, 9.2, 9.0]
```

Comment:
I'd suggest keeping plural "gains" and "noise_figures" here since in all other cases where the associated value type is an array, we've used plural keys.